src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -79,8 +79,8 @@
 # Unreserved Keywords
 bigquery_dialect.sets("unreserved_keywords").add("SYSTEM_TIME")
 bigquery_dialect.sets("unreserved_keywords").remove("FOR")
-bigquery_dialect.sets("unreserved_keywords").add("STRUCT")
```

Comment:
I had to make this change to fix an existing test that was broken by this change:
```
test/dialects/dialects_test.py::test__dialect__base_file_parse[bigquery-select_struct.sql]
```